.github/workflows/nodejs.yml

```diff
@@ -104,7 +104,7 @@ jobs:
   build:
     name: Build packages
     runs-on: ubuntu-latest
-    needs: install
+    needs: [install, lints]
```

Comment:
do we need to lint first?

Reply:
My thinking is why wast time building if the lints have failed?